matlab/test/arrow/array/tArrayDisplay.m

```diff
@@ -0,0 +1,303 @@
+            fullClassName = string(class(array)); 
+            displayName = extractAfter(fullClassName, "arrow.array.");
+            classNameLink = makeLinkString(FullClassName=fullClassName, ...
+                                           ClassName=displayName, ...
+                                           BoldFont=true);
+
+            numElementString = getNumString(3);
+            numNullString = getNumString(1);
+            header = compose("  %s with %s elements and %s null value:" + newline, ...
+                classNameLink, numElementString, numNullString);
+
+            body = "    " + ArrayWithOneNull.String + newline + newline;
+            expectedDisplay = char(strjoin([header body], newline));
+            actualDisplay = evalc('disp(array)');
+            testCase.verifyEqual(actualDisplay, expectedDisplay);
+        end
+
+        function DisplayArrayWithMultipleNulls(testCase, ArrayWithMultipleNulls)
+            import arrow.internal.test.display.makeLinkString
+
+            array = ArrayWithMultipleNulls.Array;
+            fullClassName = string(class(array)); 
+            displayName = extractAfter(fullClassName, "arrow.array.");
+            classNameLink = makeLinkString(FullClassName=fullClassName, ...
+                                           ClassName=displayName, ...
+                                           BoldFont=true);
+
+            numElementString = getNumString(3);
+            numNullString = getNumString(2);
+            header = compose("  %s with %s elements and %s null values:" + newline, ...
+                classNameLink, numElementString, numNullString);
+
+
+            body = "    " + ArrayWithMultipleNulls.String + newline + newline;
+            expectedDisplay = char(strjoin([header body], newline));
+            actualDisplay = evalc('disp(array)');
+            testCase.verifyEqual(actualDisplay, expectedDisplay);
+        end
+
+        function DisplayWindowSizeThree(testCase)
+            import arrow.internal.test.display.makeLinkString
+            array = arrow.array(1:8);
+            fullClassName = string(class(array)); 
+            displayName = extractAfter(fullClassName, "arrow.array.");
+            classNameLink = makeLinkString(FullClassName=fullClassName, ...
+                                           ClassName=displayName, ...
+                                           BoldFont=true);
+
+            numElementString = getNumString(8);
+            numNullString = getNumString(0);
+            header = compose("  %s with %s elements and %s null values:" + newline, ...
+                classNameLink, numElementString, numNullString);
+
+            body = "    1 | 2 | 3 | ... | 6 | 7 | 8" + newline + newline;
+            expectedDisplay = char(strjoin([header body], newline));
+            actualDisplay = evalc('disp(array)');
+            testCase.verifyEqual(actualDisplay, expectedDisplay);
+
+        end
+
+        function DisplayStructArray(testCase)
```

Comment:
Since `ListArray` was recently added, it would be great if you could add a test case for `ListArray` too, if possible.

Reply:
Done!